example.env

```diff
@@ -4,3 +4,5 @@ DB_CONNECTION_STRING=mongodb://localhost:27017/openkudos
 DB_NAME=kudos_test
 CLIENT_ID=1111111111111.11111111111
 CLIENT_SECRET=jk4k44j5j454345k43kkkj5433j4k5j3k
+POE_API_TOKEN=jk4k44j5j454345k43kkkj5433j4k5j3k
+POE_PROJECT_ID=111111
```

Comment:
I think we can commit real project id ;)

Reply:
It is an example file so other devs will know with of env are necessary to run the project